deploy-board/deploy_board/webapp/cluster_urls.py

```diff
@@ -34,6 +35,8 @@
     url(r'^clouds/get_base_images/$', cluster_view.get_base_images_by_name),
     url(r'^clouds/get_base_images/(?P<name>[a-zA-Z0-9\-_\.]+)/$', cluster_view.get_base_images_by_name_json),
     url(r'^clouds/get_base_image_info/(?P<name>[a-zA-Z0-9\-_\.]+)/$', cluster_view.get_base_images_by_name_json),
+    url(r'^clouds/demote_image/(?P<image_id>[a-zA-Z0-9\-_\.]+)/$', cluster_view.demote_image),
+    url(r'^clouds/promote_image/(?P<image_id>[a-zA-Z0-9\-_\.]+)/$', cluster_view.promote_image),
 
```

Comment:
There should be a Cancel button as well. It should show when there is any ongoing update events.